### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by nirinchev**
These are copy-pasted from the .NET templates.

### CODE REVIEW

**Review by nielsenko — APPROVED** [review]
Looks good in realm-dotnet, so LGTM

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      label: Repro steps
+      description: If you can reproduce the issue, please provide detailed repro steps here or provide a link to a repro project.
+      placeholder: 1. X, 2. Y, 3. Crash!
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: What version of the dart/Flutter SDK are you using?
```
Comment (blagoev) [inline .github/ISSUE_TEMPLATE/bug.yml:31]:
```suggestion
      description: What version of the Dart/Flutter SDK are you using?
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    validations:
+      required: true
+  - type: dropdown
+    id: flavour
+    attributes:
+      label: What SDK flavour are you using?
```
Comment (blagoev) [inline .github/ISSUE_TEMPLATE/bug.yml:37]:
```suggestion
      label: What Realm SDK flavor are you using?
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        - MongoDB Atlas (i.e. Sync, auth, functions)
+        - Local Database only
+    validations:
+      required: true
+  - type: dropdown
+    id: app-type
+    attributes:
+      label: What type of application is this?
+      options:
+        - Dart
```
Comment (blagoev) [inline .github/ISSUE_TEMPLATE/bug.yml:48]:
```suggestion
        - Flutter Application
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        - Flutter
```
Comment (blagoev) [inline .github/ISSUE_TEMPLATE/bug.yml:49]:
```suggestion
        - Dart standalone application
```

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
+name: Feature Request
+description: Request a new feature or enhancement
+labels: [Enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to suggest improvements to the Flutter SDK!
```
Comment (blagoev) [inline .github/ISSUE_TEMPLATE/feature.yml:8]:
```suggestion
        Thanks for taking the time to suggest improvements to the Realm Flutter and Dart SDK!
```

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 3 above)
Comment (blagoev) [reply .github/ISSUE_TEMPLATE/bug.yml:48]: If the first one is preselected then I suggest it is `Flutter`

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 2 above)
Comment (blagoev) [reply .github/ISSUE_TEMPLATE/bug.yml:37]: I don't se where we ask for the realm package version. Should we add it?

**Review by blagoev — COMMENTED** [review]
It's about right. Added some suggestion. Most importantly we don't ask about the Realm package version

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 1 above)
Comment (nirinchev) [inline .github/ISSUE_TEMPLATE/bug.yml:31]: @blagoev isn't this where we ask about the package version? Or do you mean something else?

Comment (blagoev) [reply .github/ISSUE_TEMPLATE/bug.yml]: oh, I misread that.

**Review by blagoev — APPROVED** [review]

**Review by desistefanova — APPROVED** [review]

**Pull request merged**